test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -1241,12 +1241,13 @@ protected override void GetLibrariesToPreload(List<string> libraries)
             base.GetLibrariesToPreload(libraries);
         }
 
-        private void SelectAll()
+        private void SelectAll(IEnumerable<Guid> nodes)
         {
             DynamoSelection.Instance.ClearSelection();
             foreach (var node in CurrentDynamoModel.CurrentWorkspace.Nodes)
             {
-                DynamoSelection.Instance.Selection.Add(node); 
+                if (nodes.Contains(node.GUID))
+                    DynamoSelection.Instance.Selection.Add(node); 
```

Comment:
Yah and I remember my trainer said curly brackets for single-line conditional statement is for university students :smile: